src/modules/upload/upload.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common'
+
+import { UploadController } from './upload.controller'
+import { UploadService } from './upload.service'
+
+@Module({
+  controllers: [UploadController],
+  providers: [UploadService],
+})
```

Comment:
你需要 exports 出来，否则其它 module 无法使用 
```suggestion
  providers: [UploadService],
  exports:[UploadService]
})
```